pages/blog/introducing-envelop.mdx

```diff
@@ -0,0 +1,313 @@
+import withArticle from '../../ui/blog/article';
+
+export const meta = {
+  title: 'Introducing Envelop - The GraphQL Plugin System',
+  author: 'dotan',
+  tags: ['graphql', 'server', 'node', 'plugins', 'envelop'],
+  date: '2021-07-20',
+  description:
+    'A new tool by The Guild: A powerful new way for creating fast, modern and secure GraphQL servers.',
+  thumbnail: 'https://www.envelop.dev/assets/home-communication.png',
+  image: 'https://the-guild.dev/static/shared-logos/products/envelop.svg',
+};
+
+export default withArticle({ ...meta });
+
+**Today we are super excited to share with you a new open-source library we’ve been working on for the past few months!**
+
+![Envelop](/blog-assets/envelop/definition.png)
+
+## TL;DR
+
+- Envelop aims to be **The GraphQL Plugin system** ([envelop.dev](https://www.envelop.dev/))
+- Envelop is not a GraphQL server, it's just a wrapper on top of the GraphQL engine.
+- Make “hard” **GraphQL capabilities easy** by installing **powerful plugins** ([Caching](https://github.com/dotansimha/envelop/pull/311), Tracing with [Prometheus](https://www.envelop.dev/plugins/use-prometheus)/DataDog/[NewRelic](https://www.envelop.dev/plugins/use-newrelic)/[Sentry](https://www.envelop.dev/plugins/use-sentry)/[OpenTelemetry](https://www.envelop.dev/plugins/use-open-telemetry)/[ApolloTracing](https://www.envelop.dev/plugins/use-apollo-tracing), [Loggers](https://www.envelop.dev/plugins/use-logger), [GraphQL-Jit](https://www.envelop.dev/plugins/use-graphql-jit), [Persisted Operations](https://www.envelop.dev/plugins/use-persisted-operations), Security with [rate-limit](https://www.envelop.dev/plugins/use-rate-limiter)/[depth-limit](https://www.envelop.dev/plugins/use-depth-limit)/[Auth0](https://www.envelop.dev/plugins/use-auth0) and many others from the [Plugins Hub](https://www.envelop.dev/plugins))
+- **Solve once and share across the ecosystem** - Each plugin works with any HTTP server or deployment (Express/Fastify/Netlify/VercelAWS Lambda/Azure Functions/Cloudflare Workers/Google Cloud Functions) and any schema builder (SDL, Apollo Federation, Nexus, TypeGraphQL and others)
+- **Framework for Frameworks** - Envelop will become the new basis for GraphQL Frameworks. [It's already availble if you are using RedwoodJS](https://community.redwoodjs.com/t/using-graphql-envelop-helix-in-redwood-v0-35/2276), and we have PRs open for Loopback, NestJS, Parse and others.
+- **"Babel for GraphQL"** - Envelop also aims to be the "enrichment layer" for GraphQL. You can use any new GraphQL Features today (@defer/@stream, @live queries, OneOf and any open RFC already today, even if graphql-js has not yet implemented or released it)
+
```

Comment:
Is it really true? I think `@defer/@stream` requires `graphql@defer-stream-tag` release

Reply:
Defer/stream is wrong. Envelop is compatible with it though and we have an example setup with helix

Reply:
I would leave it as is.  when people will want to use it, they can see how to do it and it is possible.
For an announcement blog post I think it's ok